docs/source-2.0/spec/idl.rst

```diff
@@ -345,21 +346,21 @@ support a version greater than or equal to ``2.0`` and less than ``3.0``:
 
 A minor version SHOULD be provided when a model depends on a feature released
 in a minor update of the specification. The following example sets the
-version requirement of a file to ``2.0``, meaning that tooling MUST support a
-version greater than or equal to ``2.0`` and less than ``3.0``:
+version requirement of a file to ``2.1``, meaning that tooling MUST support a
```

Comment:
Why change this one? 2.0 was accurate too, no?

Reply:
The corresponding section in the 1.0 docs used `1.1` as an example, and since this section is discussing minor version support, I felt like this made sense.